web/react/components/popover_list_members.jsx

```diff
@@ -27,12 +43,62 @@ export default class PopoverListMembers extends React.Component {
             }
         };
     }
+
+    handleShowDirectChannel(teammate, e) {
```

Comment:
This is duplicate code from the direct messages modal. This should be consolidated into one function called from both places.